pages/glossary/matching-characters.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Matching characters
+key: matching-characters
+---
+
+A sequence of [characters](#character) is considered to match another if, after removing leading and trailing [space characters](https://www.w3.org/TR/html/infrastructure.html#space-characters) and replacing remaining occurrences of one or more space characters with a single space, the two sequences of characters are equal character-by-character, ignoring any differences in letter casing.
```

Comment:
`character-by-character` why is this hypenated?

Reply:
Fixed